src/cloudai/reporter.py

```
@@ -207,6 +208,135 @@ def report_best_dse_config(self):
+        sweep_summaries = [self._test_run_summary(sweep) for sweep in sweeps]
+        summary = {
+            "name": tr.name,
+            "status": self._scenario_status(sweep_summaries),
+            "output_path": self._relative_path(self.results_root / tr.name),
+            "artifacts": self._artifacts_excluding(
+                self.results_root / tr.name, [sweep.output_path for sweep in sweeps]
+            ),
+            "metrics": {},
+            "sweeps": sweep_summaries,
+        }
+        return summary
```

Ruff (0.15.21)

[warning] Unnecessary assignment to `summary` before `return` statement

Remove unnecessary assignment

(RET504)